taxi/views.py

```diff
@@ -32,10 +38,27 @@ def index(request):
 
 class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     model = Manufacturer
+    queryset = Manufacturer.objects.all()
     context_object_name = "manufacturer_list"
     template_name = "taxi/manufacturer_list.html"
     paginate_by = 5
 
+    def get_context_data(self, *, object_list=None, **kwargs):
+        context = super(ManufacturerListView, self).get_context_data(**kwargs)
+        name = self.request.GET.get("name", "")
+        context["search_form"] = ManufacturerSearchForm(
+            initial={"name": name}
+        )
+        return context
+
+    def get_queryset(self):
+        form = ManufacturerSearchForm(self.request.GET)
+        if form.is_valid():
+            return (self.queryset.filter
+                    (name__icontains=form.cleaned_data["name"])
+                    )
+        return self.queryset
```

Comment:
why you use both default and overriden one, leave one. Also don't forget to remove model field which is unnecessary since you add gueryset.